src/wp-includes/rest-api/endpoints/class-wp-rest-edit-site-export-controller.php

```diff
@@ -53,7 +53,9 @@ public function register_routes() {
 	 * @return WP_Error|true True if the request has access, or WP_Error object.
 	 */
 	public function permissions_check() {
-		if ( current_user_can( 'edit_theme_options' ) ) {
+		$allowed_user_capability = apply_filters( 'allowed_user_role_to_export_theme', 'export' );
+
+		if ( current_user_can( $allowed_user_capability ) ) {
```

Comment:
```suggestion
		if ( current_user_can( 'export' ) ) {
```

Rather than use a filter, I think it would be best to use the `export` capability.

Reply:
@Rajinsharwar are you available to update the PR accordingly? Thanks!